DESCRIPTION

```diff
@@ -20,7 +20,9 @@ Imports:
     mlr3,
     mlr3misc,
     paradox,
-    R6
+    R6,
+    blockCV,
```

Comment:
This should go in "Suggests", as it is not required by all methods. And then use `blockCV::` internally.